docs/orphaned_and_conflicting.md

```diff
@@ -99,7 +99,7 @@ A conflicting page might need more work as content may have been moved/rewritten
 
 With the automated PRs workflow, it is now easier to "catch" orphaned or conflicting pages before they fill the stack of pages to be removed in the repository.
 
-The process stays the same as above (using `yarn content delete bla/bla locale --redirect foo/foo` most of the time). However, the changes should be made on the branch created by the bot rather than on a new PR (checkout the existing branch rather than creating your own on your fork).
+The process stays the same as above (using `npm run content delete bla/bla locale --redirect foo/foo` most of the time). However, the changes should be made on the branch created by the bot rather than on a new PR (checkout the existing branch rather than creating your own on your fork).
```

Comment:
```suggestion
The process stays the same as above (using `npm run content delete bla/bla locale -- --redirect foo/foo` most of the time). However, the changes should be made on the branch created by the bot rather than on a new PR (checkout the existing branch rather than creating your own on your fork).
```